portfolio/src/main/webapp/script.js

```diff
@@ -64,26 +64,52 @@ function filterCourseDisplay(courseType = 'course') {
  * Fetch comments from server and display on DOM.
  */
 function loadComments() {
-  const max = document.getElementById('max-num-comments').value;
-  const url = max == 'all' ? '/list-comments' : '/list-comments?max-comments=' + max;
+  const max = parseInt(document.getElementById('max-num-comments').value) || Number.MAX_VALUE;
+  const sortOrder = document.getElementById('sort-order').value;
+  let url = '/list-comments?sort-order=' + sortOrder;
   fetch(url)
     .then(response => response.json())
     .then((data) => {
-      const display = document.getElementById('comment-container');
-      display.innerHTML = '';
-      for (let comment of data) {
-        const likeButton = createLikeButton();
-        likeButton.addEventListener('click', () => sendLike(comment));
-
-        const deleteButton = createDeleteButton();
-        deleteButton.addEventListener('click', () => deleteComment(comment));
-
-        const commentElement = createCommentElement(comment);
-        commentElement.appendChild(likeButton);
-        commentElement.appendChild(deleteButton);
-        display.appendChild(commentElement);
-      }
-  });
+      showNextNComments(data, 0, max);
```

Comment:
Lets add a param tag for /* startInd= */ 0

Reply:
Do we want one for all three args or just startInd?

Reply:
Just the startInd as it's more obscure as to what it is. Another nit: make variable max more descriptive so it's clear this is the max number of comments to show.